pkgs/top-level/all-packages.nix

```diff
@@ -4968,6 +4968,8 @@ with pkgs;
 
   e2fsprogs = callPackage ../tools/filesystems/e2fsprogs { };
 
+  fuse2fs = if stdenv.isLinux then e2fsprogs.fuse2fs else null;
```

Comment:
Ofborg does not like that...

```
nix-env failed:
warning: ignoring the user-specified setting 'restrict-eval', because it is a restricted setting and you are not a trusted user
error: while evaluating anonymous function at /var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/mr-est/ofborg-evaluator-0/.gc-of-borg-outpaths.nix:47:12, called from undefined position:
while evaluating anonymous function at /var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/mr-est/ofborg-evaluator-0/pkgs/top-level/release-lib.nix:123:6, called from /var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/mr-est/ofborg-evaluator-0/lib/attrsets.nix:414:43:
while evaluating 'hydraJob' at /var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/mr-est/ofborg-evaluator-0/lib/customisation.nix:171:14, called from /var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/mr-est/ofborg-evaluator-0/pkgs/top-level/release-lib.nix:123:14:
while evaluating the attribute 'name' at /var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/mr-est/ofborg-evaluator-0/lib/customisation.nix:176:24:
value is null while a set was expected, at /var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/mr-est/ofborg-evaluator-0/lib/customisation.nix:176:10
```

Reply:
wonder how i should do this then

Reply:
maybe just remove that since search.nixos.org nowadays shows the outputs https://search.nixos.org/packages?channel=unstable&from=0&size=50&sort=relevance&type=packages&query=e2fsprogs

Reply:
opted for an alias